app/views/agents/rdvs/show.html.slim

```diff
@@ -11,8 +11,11 @@ p.card-text
 
 .row.mb-5
   .col.text-left
-    = link_to "Supprimer", organisation_rdv_path(@rdv.organisation, @rdv, callback_path: params[:callback_path]), method: :delete, class: "btn btn-outline-danger", data: { confirm: "Avant de supprimer ce rendez-vous, assurez-vous d’avoir averti #{@rdv.users.map(&:full_name).to_sentence}. Êtes-vous sûr de vouloir supprimer ce rendez-vous ?"}
+    = link_to "Supprimer", organisation_rdv_path(@rdv.organisation, @rdv, callback_path: params[:callback_path]), method: :delete, class: "btn btn-outline-danger", data: { confirm: "Êtes-vous sûr de vouloir supprimer ce rendez-vous ? Cette action est définitive et aucune notification ne sera envoyée à #{@rdv.users.map(&:full_name).to_sentence}."}
+
   .col.text-right
+    - if @rdv.cancellable?
+      = link_to "Annuler", organisation_rdv_path(@rdv.organisation, @rdv, status: :cancel, callback_path: params[:callback_path]), method: :put, class: "btn btn-outline-danger mr-1", data: { confirm: "Êtes-vous sûr de vouloir annuler ce rendez-vous ? L'usager recevra une notification de cette annulation."}
```

Comment:
User wise I'm not sure the UX is perfect as such but it may be ok for now and open for a next ticket. 

I'd feel it'd be clearer if there was a single 'cancel' CTA, that would then open a modal and ask you if you want to notify the user or not with a checkbox.

Reply:
I think this has been discussed between Benoit and Aude. I proposed this solution as a first step that could be improve with agents tests.

Reply:
yes 👍 what you did is good enough for now